internal/cli/adv2new/adv2new.go

```diff
@@ -0,0 +1,25 @@
+package adv2new
+
+import (
+	"github.com/mongodb-labs/atlas-cli-plugin-terraform/internal/cli"
+	"github.com/mongodb-labs/atlas-cli-plugin-terraform/internal/convert"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	o := &cli.BaseOpts{
+		Fs:      afero.NewOsFs(),
+		Convert: convert.AdvancedClusterToNew,
+	}
+	cmd := &cobra.Command{
+		Use:   "advancedClusterToNew",
+		Short: "Convert advanced_cluster from provider version 1 to 2",
```

Comment:
```suggestion
		Short: "Converts advanced_cluster from provider version 1 to 2",
```

third person? not sure, so ok to also wait for docs

Reply:
cool!